tools/checked-c-convert/README.md

```diff
@@ -17,8 +17,16 @@ manually specified files with a manually specified command line, or, using
 a `compile_commands.json` compilation database. 
 
 ### Command Line
+Invoke `checked-c-convert` on the command line as:
+
+`$ checked-c-convert -p path/to/directory/containing/compile_commands.json path/to/source/file/to/convert.c`
 
 ### `compile_commands.json` database
+CMake's configuration tool can provide a `compile_commands.json` file listing the instructions for the compiler during build. Produce this with `-DCMAKE_EXPORT_COMPILE_COMMANDS=ON` added to the configuration call to cmake.
+
```

Comment:
I have a wording suggestion. I think it'd be more precise to say that "CMake's configuration tool can generate".   I think it'd also e helpful to note that as of CMake 3.12, this is only supported by the Makefile and Ninja generators for CMake (https://cmake.org/cmake/help/v3.12/variable/CMAKE_EXPORT_COMPILE_COMMANDS.html).  That way no one makes the mistake of trying to using this with an unsupported generator.